pkg/source/image.go

```diff
@@ -139,28 +146,54 @@ func (i *Image) getDesiredPodApplyConfig(bundle *rukpakv1alpha2.BundleDeployment
 				WithImage(i.UnpackImage).
 				WithImagePullPolicy(corev1.PullIfNotPresent).
 				WithCommand("/cp", "-Rv", "/unpack", "/util/bin/unpack").
-				WithVolumeMounts(applyconfigurationcorev1.VolumeMount().
-					WithName("util").
-					WithMountPath("/util/bin"),
-				).
+				WithVolumeMounts(func() []*applyconfigurationcorev1.VolumeMountApplyConfiguration {
+					var volumeMounts []*applyconfigurationcorev1.VolumeMountApplyConfiguration
+					if gocoverdirEnv != "" {
+						volumeMounts = append(volumeMounts, applyconfigurationcorev1.VolumeMount().
+							WithName("test-coverage").
+							WithMountPath(gocoverdirEnv))
+					}
```

Comment:
I don't think we need coverage in this container, right? There are no instrumented binaries here.

Reply:
That's right, init container doesn't need it. Reverted back.